app/src/main/java/com/scouts/kitchenplaner/datalayer/repositories/ProjectRepository.kt

```diff
@@ -31,4 +36,25 @@ class ProjectRepository @Inject constructor(
             meals = project.meals.map { MealEntity(it, 0) },
             allergens = project.allergenPersons.map { it.toDataLayerEntity(project.id) })
     }
+
+    suspend fun getProject(projectName: String) : Project {
+        val entity = projectDAO.getProject(projectName)
+        return Project(entity.id, entity.name, entity.startDate, entity.endDate, listOf(), listOf(), Uri.parse(entity.imageUri))
```

Comment:
In general a project should have allergen Persons and recipes and meals. Also the default implementation is listOf().